Print_even_numbers.py

```diff
@@ -0,0 +1,9 @@
+#Print Even numbers
```

Comment:
This program is not working as expected. Please correct.

<img width="660" alt="image" src="https://user-images.githubusercontent.com/38807024/200740131-ea8ed171-8e32-4018-b432-8d4a36887970.png">